src/components/ListboxGrid/distribute-resources.ts

```diff
@@ -0,0 +1,16 @@
+import { IListboxResource } from '../../hooks/types';
+
+const distributeResources = (resources: IListboxResource[], columnCount: number) => {
+  let firstColumn;
+  let restOfColumns;
+  if (columnCount === 1) {
+    firstColumn = resources;
+  } else {
+    const sliceIndex = 1 - columnCount;
+    firstColumn = resources.slice(0, sliceIndex);
+    restOfColumns = resources.slice(sliceIndex);
```

Comment:
Shorter:
```
const [firstColumn, ...restOfColumns] = resources;
```

Reply:
Not applicable as we discussed since firstColumn could have more than one item (Listbox).